.github/workflows/release.yml

```diff
@@ -0,0 +1,44 @@
+---
+name: "Create new version tag"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Version'
+        required: true
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions: 
+      contents: write
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+          ref: release
+      - name: Set up JDK 11
+        uses: actions/setup-java@v3
+        with:
+          java-version: '11'
+          distribution: 'adopt'
+          cache: maven
+      - name: setup git config
+        run: |
+          git config user.name "GitHub Actions Bot"
+          git config user.email "<>"
+      - name: Merge changes from main
+        run: |
+          git merge origin/main
+      - name: Adjust version
```

Comment:
After the merge we should run a `mvn clean package` so that the tests are executed.
Ensures that we do not release a state that is broken.